### DISCUSSION TITLE
Just an Idea: Implement sourceDateMargin in source code

### DISCUSSION THREAD

**Discussion by CavasoftIta** [codefori/vscode-ibmi#2471] [Ideas]
It would be useful to have the possibility of having in the source of a source member the date of the row displayed at the right end of the row itself

**Comment by sebjulliand** [comment]
Like the `Source Dates in Gutter` setting? (it's on the left though)
https://codefori.github.io/docs/developing/sourcedates/#source-date-settings

**Reply by CavasoftIta** [reply]
Yes like Source Dates in Gutter but not on the left but on the right of the source rows. (In my opinion, it turns out to be more effective and clearer for an IBM i programmer who is accustomed to the old SEU.)
